CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - pipe
  - splitAt/asyncSplitAt
  - multiPartition/asyncMultiPartition
+ - collate, asyncCollate: Formerly named merge. No longer takes a helper. Default with no params is now interleaving
```

Comment:
I don't think this PR includes any change on merge or collate

Reply:
Oops, that's a bit of the bigger PR that snuck into this one